docs: fixes the `epidatpy` code samples and improves formatting on the Geographic Codes page (#1713)
cmu-delphi/delphi-epidata: docs/api/geographic_codes.md

| Code | Name |
|---|---|
| `nat` | United States (National) |

{: .note}

> > **Note:** All endpoints that contain US data use `nat` to access it, whereas [`ght`](ght.md) and other endpoints that contain data for countries in the Americas use `us`.

Reviewer: suggestion: workding change. for clarity. Please double-check that the new version is what you meant to say.

```suggestion
> **Note:** All endpoints that contain US only data use `nat` to access it, whereas [`ght`](ght.md) and other endpoints that contain data for multiple countries use `us`.
```

Author: I expanded a little on your suggestion like this: 
```suggestion
> **Note:** All endpoints that contain US-only data use `nat` to access national-level data, whereas [`ght`](ght.md) and other endpoints that contain data for multiple countries use `us`.
```
